<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenetreAccueil</class>
 <widget class="QMainWindow" name="fenetreAccueil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MindPass</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ressources/MindPass-icone.png</normaloff>../ressources/MindPass-icone.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>168</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Se connecter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_id">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>148</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse email</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_mdp">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>148</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Mot de passe</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_id">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>390</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mdp">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>390</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>500</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../ressources/MindPass-transparent.svg</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>38</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionFermer"/>
   </widget>
   <widget class="QMenu" name="menuOutils">
    <property name="title">
     <string>Outils</string>
    </property>
    <widget class="QMenu" name="menuChanger_de_langue">
     <property name="title">
      <string>Changer de langue</string>
     </property>
     <addaction name="actionFrancais"/>
     <addaction name="actionEnglish"/>
    </widget>
    <addaction name="menuChanger_de_langue"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionObtenir_de_l_aide"/>
    <addaction name="separator"/>
    <addaction name="actionA_propos_de_MindPass"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuOutils"/>
   <addaction name="menuAide"/>
  </widget>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionFrancais">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Français</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionObtenir_de_l_aide">
   <property name="text">
    <string>Obtenir de l'aide</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionA_propos_de_MindPass">
   <property name="text">
    <string>À propos de MindPass</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Préférences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionFermer</sender>
   <signal>triggered()</signal>
   <receiver>fenetreAccueil</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
